frontend/web/components/feature-page/FeatureNavTab/FeatureAnalytics/components/FeatureAnalyticsChart.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useMemo } from 'react'
+import {
+  Bar,
+  BarChart,
+  CartesianGrid,
+  ResponsiveContainer,
+  Tooltip as RechartsTooltip,
+  XAxis,
+  YAxis,
+} from 'recharts'
+import Button from 'components/base/forms/Button'
+import { FlagAnalyticsData } from 'components/feature-page/FeatureNavTab/FeatureAnalytics/FeatureAnalytics.container'
+
+interface FeatureAnalyticsChartProps {
+  usageData?: FlagAnalyticsData[]
+}
+
+const FeatureAnalyticsChart: React.FC<FeatureAnalyticsChartProps> = ({
```

Comment:
This is `drawChart` that I extracted in order to memoize, otherwise it was re-calculating on every tab navigation.

Mentioning in case of perf downgrade (but should not):
It is still calculating too early: on modal opening and closing but this requires to refactor `CreateFlag` in a way that just the selected tab component is rendered